src/XrdCl/XrdClZipArchiveReader.cc

```diff
@@ -243,7 +243,7 @@ class ZipArchiveReaderImpl
     {
       delete pBuffer;
       ClearRecords();
-      pArchive.Close();
+      XRootDStatus st = pArchive.Close();
```

Comment:
There is an attribute for this situation added in C++17; is there a pre-existing `gnu:` attribute that we could use?